src/main/java/com/amazon/ion/impl/bin/IonEncoder_1_1.java

```diff
@@ -484,10 +477,8 @@ private static int writeInlineText(WriteBuffer buffer, String value, IonType typ
     /**
      * Writes an interned Symbol's address to the given WriteBuffer using the Ion 1.1 encoding for Ion Symbols.
      * @return the number of bytes written
-     *
-     * TODO: Do we need to support Symbol Addresses greater than Long.MAX_VALUE?
      */
-    public static int writeSymbolValue(WriteBuffer buffer, long value) {
+    public static int writeSymbolValue(WriteBuffer buffer, int value) {
```

Comment:
🗺️ Since Java standard library collections track their size using an `int`, SIDs are effectively limited to those that can be represented in an `int`.